src/analysis/normed_space/lp_space.lean

```diff
@@ -629,6 +630,53 @@ end
 
 end normed_space
 
+section normed_star_group
+
+variables [Π (i : α), star_add_monoid (E i)] [Π i, normed_star_group (E i)]
+
+lemma _root_.mem_ℓp.star_mem {f : Π i, E i}
+  (hf : mem_ℓp f p) : mem_ℓp (star f) p :=
+begin
+  rcases p.trichotomy with rfl | rfl | hp,
+  { apply mem_ℓp_zero,
+    simp [hf.finite_dsupport] },
+  { apply mem_ℓp_infty,
+    simpa using hf.bdd_above },
+  { apply mem_ℓp_gen,
+    simpa using hf.summable hp },
+end
+
+@[simp] lemma _root_.mem_ℓp.star_iff {f : Π i, E i} : mem_ℓp (star f) p ↔ mem_ℓp f p :=
+⟨λ h, star_star f ▸ mem_ℓp.star_mem h ,mem_ℓp.star_mem⟩
```

Comment:
We cannot use dot-notation here, so I thought mathlib prefers the name `_root_.mem_ℓp_star_iff`?

Reply:
We should match whatever the `neg_iff` lemma above does

Reply:
Which it seems already matches this new lemma; so I think best to leave things as is

Reply:
bors r+